learning_resources/views_learningpath_test.py

```diff
@@ -370,12 +370,11 @@ def test_get_resource_learning_paths(user_client, user, is_editor):
         factories.LearningPathRelationshipFactory.create_batch(
             3, child=course.learning_resource
         ),
-        key=lambda item: item.position,
+        key=lambda item: item.id,
```

Comment:
The view for LearningPathRelationships specifies
```py
    filter_backends = [OrderingFilter]
    ordering = ["position", "-child__last_modified"]
```

> If an ordering attribute is set on the view, this will be used as the default ordering.
> – [drf docs](https://www.django-rest-framework.org/api-guide/filtering/)

The "relationships" are re-orderable, and the API is intended to, by default, return the relationships ordered by position.

Reply:
good callout. i'm going to put this back in draft and have a deeper look

Reply:
My original comment was inaccurate: I did not look closely enough and misunderstood which API was being tested here.
-  My comment applies to `/api/v1/learningpaths/{LIST_ID}/items/`
- This API is testing `/api/v1/courses/{COURSE_ID}`. *Courses include a "micro" list of the items. *

So I think your original solution is fine.